refugees-stories/frontend
# Add user registration

### What does this PR do?
Adds  the functionality for new users to register

### Description of Task to be completed?
Create form and logic for new users to register


### How should this be manually tested?
Clone the repo, run yarn start to start up the server.
cd into refugees-stories, on the terminal do yarn install and after installation of npm packages run yarn start to start up the application. Navigate to http://localhost:3000/register. If successful, for now, you should get status code of `201` in the console.
